### ISSUE TITLE
MentaLiST conda fails when using more recent julia

### ISSUE THREAD

**Issue by apetkau** [WGS-TB/MentaLiST#80]
I was recently installing MentaLiST through conda and when I tried to run `mentalist` I got the following error:

```
ERROR: LoadError: ArgumentError: Package Lumberjack not found in current path:
- Run `Pkg.add("Lumberjack")` to install the Lumberjack package.

Stacktrace:
 [1] require(::Module, ::Symbol) at ./loading.jl:817
 [2] include at ./boot.jl:317 [inlined]
 [3] include_relative(::Module, ::String) at ./loading.jl:1038
 [4] include(::Module, ::String) at ./sysimg.jl:29
 [5] exec_options(::Base.JLOptions) at ./client.jl:229
 [6] _start() at ./client.jl:421
in expression starting at /home/CSCScience.ca/apetkau/miniconda3/envs/mentalist-0.2.3/bin/mentalist:2
```

I believe this is related to the version of `julia` that gets installed by conda, which is now version `1.0.0` by default. If I downgrade to version `0.5.2`, then `mentalist` will work.

**Comment by dfornika** [comment]
Thanks for reporting. This is related to the Bioconda project's shift to use 'global pinning' to try to standardize dependencies across recipes. When updating the MentaLiST conda recipe to use the new 'conda-build 3' syntax, the recommendation was to not specify version numbers for dependencies in the `meta.yaml` file.

But since MentaLiST-0.1.x is developed against `julia-0.5.2` and MentaLiST-0.2.x is developed against `julia-0.6.1`, we do need to pin against those conda packages as dependencies.

I have an open pull-request in the `bioconda-recipes` repo attempting to fix this:

https://github.com/bioconda/bioconda-recipes/pull/10720

...but it hasn't been merged yet.

**Comment by apetkau** [comment]
Thanks. That makes sense. I'm glad to see you already have a fix, even before I reported :smile:

**Comment by dfornika** [comment]
Having a bit of trouble getting attention on that pull-request. I haven't forgotten about this. I'll keep trying.

**Comment by dfornika** [comment]
The pull request has been merged. The fixed build should be `mentalist-0.1.9-h<xxxxxxx>_3` (where `<xxxxxxx>` is some hexadecimal build id that I don't completely understand)

**Comment by apetkau** [comment]
Thanks @dfornika, this is working great now.
